Directory.Build.props

```diff
@@ -8,21 +9,9 @@
     <DebugType>embedded</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <LangVersion>Latest</LangVersion>
-    <IsPackable>true</IsPackable>
     <EnablePackageValidation>true</EnablePackageValidation>
 
-    <CommonPath>$(MSBuildThisFileDirectory)src/Common/</CommonPath>
```

Comment:
Do forward slashes give trouble in Windows?

Reply:
No but they appear weird in a binlog. The NormalizeDirectory/NormalizePath msbuild functions use the correct slashes.